src/odemis/dataio/tiff.py

```diff
@@ -1208,14 +1208,15 @@ def _findImageGroups(das):
 
     for da in das:
         # check if it can be part of the current group (compare just to the previous DA)
+        dims = da.metadata.get(model.MD_DIMS, "CTZYX"[-da.ndim::])
         if (prev_da is None
-            or da.shape[0] != 1  # If C != 1 => not possible to merge (C is always first dimension)
+            or ("C" in dims and da.shape[dims.index("C")] != 1)  # If C != 1 => not possible to merge (C is always first dimension)
+            or (da.shape[-2] != 1 and len(da.shape) == 2)  # thumbnail image of format YX
```

Comment:
Is that check necessary? I thought thumbnails are not included in this "groups" anyway. I fear it's going to exclude pretty much any array with shape == 2, while it should be allowed to store FM images as greyscale YX.

Reply:
I have reverted it to the original code and only updated the comment.